go/src/meguca/parser/body.go

```diff
@@ -82,15 +79,16 @@ func ParseBody(body []byte, board string, internal bool) (
 				links = append(links, l)
 			}
 		case '#':
-			if body[lineStart] == '>' { // Ignore hash commands in quotes
+			// Ignore hash commands in quotes, or #pyu/#pcount if board option disabled
+			if body[lineStart] == '>' || (word[1] == 'p' && !pyu) {
```

Comment:
Index out of bounds, if length of word = 1.

Reply:
I'll add a length check.